ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -3810,11 +3810,38 @@ static void ggml_backend_sycl_graph_compute_impl(ggml_backend_sycl_context * syc
     }
 }
 
+#ifdef GGML_SYCL_GRAPH
+static bool check_node_graph_compatibility(ggml_cgraph * cgraph) {
```

Comment:
CUDA checks if one of the buffer is split, meaning multiple devices are used, to disable the graph. I assume you don't test that, may be safe to disable it as well? Or do you expect SYCL-Graph would work well in such a case?
I'd suggest just checking if `ggml_sycl_info().device_count > 1` to disable SYCL-Graph.

Reply:
Thanks for catching that, a SYCL-Graph object can only be associated with a single device, so disabled the feature for this case as well.